sentry-android/build.gradle.kts

```diff
@@ -30,3 +32,13 @@ dependencies {
     api(project(":sentry-android-core"))
     api(project(":sentry-android-ndk"))
 }
+
+afterEvaluate {
```

Comment:
Can you factor out the repeated publishing logic into the main `build.gradle` file?